po/ar.po

```diff
@@ -24,7 +24,7 @@ msgid "3. Paste the Qr-Code here:"
 msgstr ""
 
 #: src/pages/About.vue:4 src/pages/Register.vue:6
-msgid "A cross-plattform signal client"
```

Comment:
Hey sc0w, first of all, thank you for your first contribution to the Axolotl project. Any help is appreciated. For this change here, it isn't enough to change the `msgid` because it's autogenerated from the .vue files. If you propose to change the spelling in English, we have to do at the place of the origin of the string. Here it's `src/pages/About.vue:4 src/pages/Register.vue:6` https://github.com/nanu-c/axolotl/blob/main/axolotl-web/src/pages/About.vue#L5

After that the po and pot files have to be regenerated is explained here: https://github.com/nanu-c/axolotl/blob/main/docs/TRANSLATE.md 

if you have further questions please ask.

Reply:
@nanu-c ok, I did new changes, now I did a little change, the update of pot/po files can be done by someone later into new PR